utils/common/src/main/java/org/apache/brooklyn/util/ssh/BashCommands.java

```diff
@@ -671,6 +672,15 @@ public static String installJavaLatestOrWarn() {
     }
 
     /**
+     * Adds the PPA for OpenJDK for older JDK versions (7 and lower) required by some software (e.g. JBoss)
+     */
+    public static String addOpenJDKPPK(){
```

Comment:
I would agree as will be the case in future YAML based blueprints, but this is the pattern used elsewhere, see JMX or NSS